packages/hint-button-type/README.md

```diff
@@ -0,0 +1,62 @@
+# button-type (`@hint/hint-button-type`)
+
+Check if button has type attribute set
+
+## Why is this important?
+
+The default type for <button> is "submit" (not type="button" as one might expect). This can lead to surprising keyboard behavior within a form. 
```

Comment:
We have some linting rules for `*.md` files. Just run `yarn lint:md` (or `yarn lint` to lint all file types) from this package's folder to get the feedback. One thing it should flag is keeping line length under a certain number of characters.